rollout/trafficrouting/istio/istio_test.go

```diff
@@ -3156,3 +3156,45 @@ spec:
 	actions := client.Actions()
 	assert.Len(t, actions, 0)
 }
+
+func TestGetHttpRouteIndexesToPatch(t *testing.T) {
+
+	// Test case when the rollout has no managed routes defined
+	httpRoutes := []VirtualServiceHTTPRoute{
+		{Name: "foo", Match: nil},
+		{Name: "", Match: nil},
+	}
+
+	indexes, err := getHttpRouteIndexesToPatch([]string{}, httpRoutes)
+	assert.NoError(t, err)
+	assert.Equal(t, []int{1}, indexes)
```

Comment:
Can you wrap each test inside a Run block. For instance,

```go
t.Run("the rollout has no managed routes defined, func(t *testing.T) {
	httpRoutes := []VirtualServiceHTTPRoute{
		{Name: "foo", Match: nil},
		{Name: "", Match: nil},
	}

	indexes, err := getHttpRouteIndexesToPatch([]string{}, httpRoutes)
	assert.NoError(t, err)
	assert.Equal(t, []int{1}, indexes)
}
```

Reply:
@agaudreault done.